python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -646,9 +647,10 @@ def main():
 
     try:
         import Crypto
-        print 'PyCrypto %s' % (Crypto.__version__,)
+        pycrypto_version = Crypto.__version__
```

Comment:
Instead of the compatible "Crypto", I think that we should install `pycryptodomex` and have import `import Cryptodome`

In this way, we are sure that we have PyCryptodome.
Otherwise this test will pass even when pycrypto is installed

Reply:
This is not a test, it should just output the version. The tests for PyCryptodome are in `chevah_build. PyCrypto is removed completely in this branch anyway.